cypress/e2e/data-import/permissions/cannot-stop-running-jobs-with-can-only-view-permission.cy.js

```diff
@@ -0,0 +1,59 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Users from '../../../support/fragments/users/users';
+
+describe('data-import', () => {
+  describe('Permissions', () => {
+    let userA;
+    let userB;
+    const filePath = 'marcBibFileForC356788.mrc';
+    const marcFileName = `C356788 autotestFileName ${getRandomPostfix()}`;
+    const jobProfileToRun = 'Default - Create instance and SRS MARC Bib';
+
+    before('create test data', () => {
+      cy.createTempUser([Permissions.settingsDataImportView.gui]).then((userProperties) => {
+        userA = userProperties;
+      });
+
+      cy.createTempUser([
+        Permissions.moduleDataImportEnabled.gui,
+        Permissions.settingsDataImportEnabled.gui,
+        Permissions.inventoryAll.gui,
+      ]).then((userProperties) => {
+        userB = userProperties;
+        cy.login(userB.username, userB.password, {
+          path: TopMenu.dataImportPath,
+          waiter: DataImport.waitLoading,
+        });
+      });
+    });
+
+    after('delete test data', () => {
+      cy.getAdminToken();
+      Users.deleteViaApi(userA.userId);
+      Users.deleteViaApi(userB.userId);
+    });
+
+    it(
+      'C356788 A user cannot stop running jobs with "Data import: Can view only" permission (folijet)',
```

Comment:
description should have '(TaaS)' at the end:
 C356788 A user cannot stop running jobs with "Data import: Can view only" permission (folijet) (TaaS)

Reply:
updated